modules/imgproc/src/shapedescr.cpp

```diff
@@ -879,38 +904,39 @@ static Rect pointSetBoundingRect( const Mat& points )
     if( npoints == 0 )
         return Rect();
 
-#if CV_SIMD
+#if (CV_SIMD || CV_SIMD_SCALABLE)
```

Comment:
Let's also disable this block for SCALABLE mode and revert all unnecessary modifications leaving only automatically replaced intrinsics constructs, otherwise it becomes too complex. This function should be refactored separately: first for wide intrinsics, then for scalable.

Reply:
Done
```
#if CV_SIMD // TODO: enable for CV_SIMD_SCALABLE, loop tail related.
```